tests/unit/nodes/test_transform.py

```diff
@@ -24,7 +24,7 @@ class MyData:
 
 
 @as_function_node
-def Downstream(x: MyData.dataclass):
+def Downstream(x: MyData) -> MyData:
```

Comment:
These are not the same thing. `MyData` is a node, `MyData.dataclass` is a dataclass.

Reply:
My mistake. On the first glance it just looked wrong. Good to have reviews, bad mypy did not complain about my incorrect change.